pages/1.12/release-notes/1.12.2/index.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.2
+title: Release Notes for 1.12.2
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.2
+---
+
+DC/OS Version 1.12.2 was released on February 7, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.12.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise [/button]
+
+DC/OS 1.12.2 includes the following components:
+- Apache Mesos 1.7.x [change log](https://github.com/apache/mesos/blob/b97f0ba29d40a279dec00ffe51512e3b5a146049/CHANGELOG).
+- Marathon is 1.7.x [change log](https://github.com/mesosphere/marathon/blob/48bfd6000c544df5ae03de04b42b019d5e9dbd4b/changelog.md).
+- Metronome is 0.5.71 [change log](https://github.com/dcos/metronome/blob/22945457c7cb10cb14d575ceeb137edd8158ba3c/changelog.md).
```

Comment:
link should be:  
https://github.com/dcos/metronome/blob/391637cc19cd6136e8733ff8b684aed31b2cf672/changelog.md

short SHA found here: https://github.com/dcos/dcos/blob/1.12.2/packages/metronome/buildinfo.json#L5
Full SHA found here (by searching short SHA): https://github.com/dcos/metronome/commit/391637cc19cd6136e8733ff8b684aed31b2cf672

Reply:
Changed the link.
Is there something I need to do with the SHA json link? I haven't seen that before.